l3kernel/l3debug.dtx

```diff
@@ -838,22 +838,23 @@
       \skip_set_eq:NN
       \skip_add:Nn
       \skip_sub:Nn
-      \__kernel_tl_set:Nx
+      \__kernel_tl_set:Ne
       \tl_clear:N
       \tl_set_eq:NN
       \tl_put_left:Nn
       \tl_put_left:NV
       \tl_put_left:Nv
+      \tl_put_left:Ne
       \tl_put_left:No
       \tl_put_left:Nx
       \tl_put_right:Nn
       \tl_put_right:NV
       \tl_put_right:Nv
+      \tl_put_right:Ne
       \tl_put_right:No
       \tl_put_right:Nx
       \tl_build_begin:N
       \tl_build_put_right:Nn
-      \tl_build_put_right:Nx
```

Comment:
`\tl_build_put_right:Nx` is still defined...

Reply:
Corrected: these were tricky as the list has to match the hand-tuned definitions.